bionic/flow.py

```diff
@@ -1480,10 +1478,27 @@ def ...
 
             from mymodule import flow
             ...
-            flow.reloading().get('my_entity')
+            flow.reload()
+            flow.get('my_entity')
+
+        This will update the flow instance to use reloaded the modules before
+        doing the ``get()``.
 
-        This will reload the modules and use the most recent version of the
-        flow before doing the ``get()``.
+        Entities marked with ``@changes_per_run`` will be recomputed.
```

Comment:
Oh, we probably should also mention that this resets the flow's in-memory cache. That will also give more context to this sentence, so we could say something like "Entities that have not been persisted to disk, or that have been marked with ``@changes_per_run``, will be recomputed."

Reply:
Also, now that we know that merged flows don't work, that might be a good thing to note in this docstring.

Reply:
Hmm, we already have "has never been modified (i.e., isn't derived from another Flow)" as one of the safety requirements.

Reply:
Ah, I see what you mean. That text predates the merging functionality, but was intended to refer to flows derived by calling methods on an existing flow (like `flow.setting`); i.e., you can only call `reload()` on a flow that was returned by `builder.build()`. You're right that the text can also be reasonably interpreted to disallow merging as well, but technically if you merge in a flow before calling `build()`, then the *resulting* flow object "has never been modified". So it might be worthwhile to make this text slightly more explicit.

Reply:
Added safety requirements about flow merging.